api/v1/runtimecomponent_types.go

```diff
@@ -84,6 +84,10 @@ type RuntimeComponentSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:order=12,type=spec,displayName="Probes"
 	Probes *RuntimeComponentProbes `json:"probes,omitempty"`
 
+	// TODO fix the ordering
```

Comment:
ServiceAccountName field's description (in API) should be marked as deprecated. `.spec.serviceAccount.name` overrides this if both are specified.